docs/book/SUMMARY.md

```diff
@@ -29,6 +29,7 @@
   * [Volume Topology](features/volume_topology.md)
   * [Volume Health](features/volume_health.md)
   * [vSphere CSI Migration](features/vsphere_csi_migration.md)
+  * [vSphere CSI on VMC](features/vsphere_csi_vmc.md)
```

Comment:
Should we also add this to the supported features matrix?
https://vsphere-csi-driver.sigs.k8s.io/supported_features_matrix.html

Reply:
Can we add this at the time of release?

Reply:
Yes. will update the supported_features_matrix at the time of release.